src/shogun/machine/GaussianProcessMachine.h

```diff
@@ -127,6 +127,27 @@ class GaussianProcessMachine : public Machine
 		Machine::set_labels(lab);
 		m_method->set_labels(lab);
 	}
+
+	virtual SGVector<float64_t>
+	get_mean_vector(const std::shared_ptr<Features>& data)
+	{
+		not_implemented(SOURCE_LOCATION);
+	}
```

Comment:
@karlnapf I guess we need this, as well as marginal likelihood, so cannot just have Machine as a base class in interfaces

Reply:
yes, I think so